tests/Domain/RegularGivingServiceTest.php

```diff
@@ -181,9 +181,9 @@ public function testItCreatesRegularGivingMandate(): void
         $this->assertEquals(Country::fromEnum(CountryAlpha2::Kiribati)->alpha2->value, $this->donorAccount->getBillingCountryCode());
         $this->assertSame('KI0107', $this->donorAccount->getBillingPostcode());
 
-        $this->assertSame(MandateStatus::Active, $mandate->getStatus());
-
-        $this->regularGivingNotifierProphecy->notifyNewMandateCreated(Argument::cetera())->shouldBeCalled();
+        // Pending, and no notification, until Stripe first donation charge succeeded webhook.
+        $this->assertSame(MandateStatus::Pending, $mandate->getStatus());
+        $this->regularGivingNotifierProphecy->notifyNewMandateCreated(Argument::cetera())->shouldNotBeCalled();
```

Comment:
IMHO we can delete this line, we can never assert all the things that should not be called and prophecy will assume it should not be called just by us not saying it should be called.

Reply:
Yep OK, removed in https://github.com/thebiggive/matchbot/commit/3f63f378e0610bed891780bb64e4db654f65e9c2